index.html

```diff
@@ -456,6 +456,53 @@ <h2>DID Parameters</h2>
 
 </section>
 
+<section id="did-service-types">
+	<h2>DID Service Types</h2>
+
+	<p>
+		DID Documents MAY contain <code>services</code> each of which MUST have
+		a <code>type</code>. The <code>type</code> is to define the kind of
+		service being described by the service endpoint. Such a specification
+		may define the semantics of the service, the protocol(s) to be used to
+		interact with the service, the attributes in <code>service</code>
+		object. Such specifications MAY be defined here, within a DID Method
+		Specification, or as a DID Extensions. DID Resolvers SHOULD support the
+		service <code>type</code> values defined here, plus any others defined
+		elsewhere.
+	</p>
+	<section id="pathservice-service-type">
+		<h3><code>pathService</code> Service Type</h3>
```

Comment:
```suggestion
		<h3><code>PathService</code> Service Type</h3>
```

I believe is the convention